### DISCUSSION TITLE
Import quotations

### DISCUSSION THREAD

**Discussion by s2b-git** [YetiForceCompany/YetiForceCRM#16641] [How to]
I stuck on importing quotations to latest YF. Whatever value I set in Tax Mode field (source csv) it returning me following:
2022-11-09 12:41:50.6062 [error][] - ERR_ILLEGAL_FIELD_VALUE||taxmode||
App\Exceptions\Security: ERR_ILLEGAL_FIELD_VALUE||taxmode|| in /var/www/html/modules/Vtiger/inventoryfields/TaxMode.php:41

Is there any trick on that?

Best

**Comment by TonyM15** [comment]
I did tried to import sales orders. the same issue. Can do one by one only. With 6k records not easy
